auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -358,14 +375,33 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
         }
       }
     }
-    const filingResponse = await this.createNamedBusiness(filingBody)
+    return this.createNamedBusiness(filingBody, business.businessIdentifier)
+  }
 
-    if (filingResponse?.errorMsg) {
-      this.$emit('add-unknown-error')
-      return ''
-    } else {
-      return filingResponse.data.filing.business.identifier
+  private async createBusinessRegistration (business: Business): Promise<any> {
+    const filingBody: RegistrationRequest = {
+      filing: {
+        header: {
+          name: FilingTypes.REGISTRATION,
+          accountId: this.currentOrganization.id
+        },
+        business: {
+          legalType: business.nameRequest.legalType,
+          natureOfBusiness: business.nameRequest.natureOfBusiness
+        },
+        registration: {
+          nameRequest: {
+            legalType: business.nameRequest.legalType,
+            nrNumber: business.businessIdentifier
+          }
+        }
+      }
+    }
+    if (business.nameRequest.legalType === LegalTypes.SP) {
```

Comment:
Add a comment to what you're doing here.

Should this use a generic method (helper / mixin ?) to return Entity Type Code when given the Legal Type? (Is this used elsewhere? Are we expecting to use this again soon?)

Reply:
It may not be required always

Reply:
OK, leave it for now.

Reply:
... But please add that comment :)